src/Time/Units.hs

```diff
@@ -116,6 +122,16 @@ instance KnownSymbol (ShowUnit unit) => Show (Time unit) where
                                                 n -> '/' : show n
                       in numeratorStr ++ denominatorStr ++ symbolVal (Proxy @(ShowUnit unit))
 
+instance KnownSymbol (ShowUnit unit) => Read (Time unit) where
+    readPrec = lift readP
+      where
+        readP = do
+            n <- munch1 isDigit
+            m <- option "1" (char '/' *> munch1 isDigit)
+            timeUnitStr <- munch1 isLetter
+            unless (timeUnitStr == symbolVal (Proxy @(ShowUnit unit))) pfail
+            pure $ Time (read n % read m)
```

Comment:
I think you do double work here. This instance can be optimized a little bit. See, you first return `1` as a string `"1"` and only then you call `read` over string `"1"`. I think we can expect that in reality users almost never will try to parse something like `13/15mcs`. Thus returning `1` as number can gain us some performance benefits. I think your instance can be improved a little bit, if you write `let naturalP = read <$> munch1 isDigit` and then you use it like this:

```haskell
m <- option 1 (char '/' *> naturalP)
```